From 3881a792bbe66765ccf527853c96da0f84ba94b2 Mon Sep 17 00:00:00 2001
From: Jonathan Helmus <jjhelmus@gmail.com>
Date: Sun, 23 Jul 2017 16:20:49 -0500
Subject: [PATCH] do not download externals

Do not download externals in build.bat script.  Downloading and
unpacking on external dependencies is done in the conda build bld.bat
script.
---
 PCbuild/build.bat | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/PCbuild/build.bat b/PCbuild/build.bat
index 8e1c168..2d602e1 100644
--- a/PCbuild/build.bat
+++ b/PCbuild/build.bat
@@ -88,7 +88,7 @@ if "%IncludeExternals%"=="" set IncludeExternals=false
 if "%IncludeSSL%"=="" set IncludeSSL=true
 if "%IncludeTkinter%"=="" set IncludeTkinter=true
 
-if "%IncludeExternals%"=="true" call "%dir%get_externals.bat"
+rem if "%IncludeExternals%"=="true" call "%dir%get_externals.bat"
 
 if "%platf%"=="x64" (
     if "%on_64_bit%"=="true" (
-- 
2.13.3.windows.1

